### DISCUSSION TITLE
Why a slider that have a lot of anchors make osu!lazer took sometime to actually show all anchors?

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#25210] [Q&A]
https://github.com/ppy/osu/assets/59832159/62eb2901-e5ef-4d0f-ac86-b08e31cc753e  

osu!lazer take quite sometime and some cpu usage spikes to actually show sliders with lots of anchor while stable have no problem at all and can show it immediately every time i hover on a slider

**Comment by peppy** [comment]
Will require some optimisation.

That said, you shouldn't need so many nodes in lazer due to the new curve modes it supports.

**Reply by timelessnesses** [reply]
these kind of sliders are called buzz slider or tumor slider and they are sometimes used in songs where it has an intense amount beats (can be replaced by slider with a lot of reverse arrows) as you can see in the video

**Reply by timelessnesses** [reply]
another example of this is leaf - marenol (https://osu.ppy.sh/beatmapsets/1136149#osu/2404722) extra difficulty also have these kind of slider

**Reply by peppy** [reply]
"Buzz" sliders or sliders with many repeats do not create more nodes like the video you provided. I think you may be a bit confused.

**Reply by timelessnesses** [reply]
> "Buzz" sliders or sliders with many repeats do not create more nodes like the video you provided. I think you may be a bit confused.

i meant these kind of buzz sliders  
![image](https://github.com/ppy/osu/assets/59832159/610800c1-4d67-4f19-8db9-32d2b31b6cf0)  

not these kind of buzz sliders   
![image](https://github.com/ppy/osu/assets/59832159/78ca5922-85ae-4b7c-82dc-7a26e5677655)

**Reply by peppy** [reply]
You're making up your own definition of terminology now. Please leave this where it is. Optimisations will happen in the future.

**Comment by timelessnesses** [comment]
sorry if i confused you
